src/orquestra/sdk/_base/cli/_dorq/_entry.py

```diff
@@ -60,6 +60,13 @@
 
 
 @cloup.group(context_settings=CLICK_CTX_SETTINGS)
+@click.version_option(
+    None,  # version number not set, click will infer it using setuptools
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏the version is inferred from [`importlib.metadata`](https://docs.python.org/3/library/importlib.metadata.html#module-importlib.metadata) not `setuptools`.

Reply:
So it is, stackoverflow misled me